Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11.4
+
+RUN apt update && apt install -y xvfb firefox-esr
+
+COPY . /app
+WORKDIR /app
+RUN python3 setup.py develop
+
+ENV PATH /usr/lib/firefox-esr:/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:
+CMD xvfb-run python -c 'from grafanimate.commands import run; run()' --header-layout=no-chrome --scenario=scenarios.py:ontario_windsolar --output=./animations --video-fps=30 --video-framerate=30
```

Comment:
We need to do something here, as we can't ship an OCI container including a specific command. However, I am struggling to generalize this, see https://github.com/panodata/grafanimate/pull/24#discussion_r1361339023. Maybe you can help?

Reply:
I just pushed dcee772 please squash this into my Add dockerfile commit that you cherry picked.

Reply:
The new Dockerfile looks like this: https://github.com/panodata/grafanimate/blob/dockerize/Dockerfile

Reply:
Works in dcee772
```
$ docker run --rm grafanimate
Error: Parameter --scenario is mandatory
Usage:
      grafanimate [options] [--target=<target>]...
      grafanimate --version
      grafanimate (-h | --help)
```
```
$ docker run --rm grafanimate --header-layout=no-chrome --scenario=scenarios.py:gb_windsolar --output=./animations --video-fps=30 --video-framerate=30 --exposure-time=0.2
2023-10-16 22:53:27,111 [grafanimate.grafana           ] INFO   : Starting GrafanaWrapper on https://grafana-backup.int.mog.se/d/e0a625dd-20a9-447a-92d8-55eef5b6b077/generation-mix-grafanimate-gb?orgId=1&kiosk=1
2023-10-16 22:53:27,111 [grafanimate.marionette        ] INFO   : Starting Marionette Gecko wrapper
2023-10-16 22:53:27,112 [grafanimate.marionette        ] INFO   : Found "firefox" program at /usr/lib/firefox-esr/firefox-esr
2023-10-16 22:53:27,112 [grafanimate.marionette        ] INFO   : Check for running instance of Marionette/Firefox at localhost:2828
2023-10-16 22:53:27,112 [grafanimate.marionette        ] INFO   : Will launch new Marionette/Firefox instance
mozversion INFO | application_buildid: 20230918121054
mozversion INFO | application_changeset: 17bedcda2f5c28b7fbc136d300bdbbf04f382e00
..............
2023-10-16 22:53:39,052 [grafanimate.grafana           ] INFO   : Timewarp to 2022-10-01 00:00:00 -> 2022-10-31 00:00:00
2023-10-16 22:53:39,134 [grafanimate.animations        ] INFO   : Waiting for 0.2 seconds (exposure time)
2023-10-16 22:53:39,335 [grafanimate.grafana           ] INFO   : Timewarp to 2022-10-01 04:00:00 -> 2022-10-31 04:00:00
2023-10-16 22:53:39,428 [grafanimate.animations        ] INFO   : Waiting for 0.2 seconds (exposure time)
```

I have not tested the results all the way... ohh and ffmpeg will be missing. But it runs!

Reply:
Does `docker run --rm grafanimate --version` work for you?

Reply:
```
$ docker run --rm -it grafanimate -- --version
grafanimate 0.7.0
```
It does with extra `--` before `--version`
This is confusing, let's move to  your PR.

Reply:
I can confirm that, but the same happens over there as well.

Also works, it does not have to be the `--` infix: ;]
```
$ docker run --rm local/grafanimate foo --version
grafanimate 0.7.0
```

Funny!